torntools/scripts/content/global/ttGlobal.js

```diff
@@ -31,6 +31,8 @@ requireDatabase().then(() => {
 	if (settings.developer) {
 		showCustomConsole();
 	}
+	
+	if (settings.pages.global.easter_eggs && new Date().getMonth() === 3 && new Date().getDate() >= 5 && new Date().getDate() <= 25) easterEggs();
 
```

Comment:
1. The event also has hours attached to it (12 TCT)
2. You are checking local time instead of UTC time.